decidim-initiatives/app/views/decidim/initiatives/initiatives/show.html.erb

```diff
@@ -202,7 +202,7 @@ edit_link(
 <section class="layout-main__section">
   <%= comments_for current_initiative if current_initiative.commentable? && current_initiative.published? %>
   <ul class="drawer-metadata__container layout-main__section" data-metadata-footer>
```

Comment:
This does not work as expected, maybe this needs to be wrapped like this?

```
  <% content_for :item_footer do %>
    <%= comments_for current_initiative if current_initiative.commentable? && current_initiative.published? %>
    <ul class="metadata__container layout-main__section" data-metadata-footer>
      <%= content_tag :li, resource_reference(current_initiative), class: "metadata__item" %>
      <%= content_tag :li, resource_version(current_initiative, versions_path: initiative_version_path(current_initiative, current_initiative.versions.count)), class: "metadata__item" %>
    </ul>
  <% end %>
```
![image](https://github.com/decidim/decidim/assets/105683/019670ed-4904-4b35-bbb0-c8cd5bd47f06)

Reply:
Done

Reply:
![image](https://github.com/decidim/decidim/assets/105683/829e36f0-1fb7-425c-b7c1-a6bc30af1033)